opm/parser/eclipse/EclipseState/AquiferCT.hpp

```diff
@@ -67,7 +84,7 @@ namespace Opm {
 
             bool operator==(const AQUCT_data& other) const;
         };
-
+        AquiferCT() = default;
         AquiferCT(const TableManager& tables, const Deck& deck);
         AquiferCT(const std::vector<AquiferCT::AQUCT_data>& data);
```

Comment:
It is not in this PR. `explicit` can be added here.

Reply:
Does it make sense to add `explicit` to a no argument constructor?

Reply:
I refer to line 89. I read the code locally on my computer and ran into it. You do not have to address it.